packages/flame_bloc/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## 0.0.1
```

Comment:
Maybe keep the same format as the flame changelog with a header etc? And if you want to release this with the script it should say [Next].